core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -53,7 +53,7 @@ class AnalyzerManager(
     case req @ RestartScalaCompilerReq(id, _) =>
       id match {
         case Some(projectId) =>
-          getOrSpawnNew(projectId) forward req
+          analyzers.getOrElse(projectId, context.system.deadLetters) forward req
```

Comment:
`analyzers.get(projectId).foreach(_ forward req)` is better, no need to get `deadLetters` involved